openlibrary/plugins/openlibrary/home.py

```diff
@@ -47,105 +46,31 @@ def GET(self):
         except:
             loans = None
 
-        return render_template(
+        page = render_template(
             "home/index", stats=stats,
             blog_posts=blog_posts,
```

Comment:
Not part of this review... but blog posts? Look into what this does and if we're incurring any cost generating the contents of this unused template var

Reply:
This is being used, in the footer. There's probably a lot more we can do with this, e.g. staff picks